setup.py

```diff
@@ -91,7 +120,8 @@ def run(self):
         # save binary to self.build_temp
         url = get_download_url()
         data = download(url)
-        save_executable(data, self.build_temp)
+        files = extract(url, data)
+        save_files(files, self.build_temp)
```

Comment:
why are we extracting more than just the exe now?

Reply:
Why not? At least the license file sounds like something that we actually should. And considering upstream has chosen to not ship the exe separately, sounds like following what they want. Anyway these are just two text files.

Reply:
well for one, it's writing those into `./bin` which is bad

Reply:
Yeah, but in a fairly cosmetic sense, they're not executable or anything.